# 3301-
Scripts for 16S rRNA amplicon analysis using QIIME
Scripts used to perform microbiome analysis of raw DNA sequencing data in FASTAQ format. FASTAQ files, along with the mapping file containing all sample metadata, were uploaded to the SURFsara high performance computer cluster Cartesius and analysed using QIIME. The first part of each script describes the cluster resources required to run to script (SBATCH) Full details of the QIIME scripts used in subsequent analysis can be found on the QIIME website (http://qiime.org). Scripts used to perform microbiome analysis of raw DNA sequencing data in FASTAQ format. FASTAQ files, along with the mapping file containing all sample metadata, were uploaded to the SURFsara high performance computer cluster Cartesius and analysed using QIIME. The first part of each script describes the cluster resources required to run to script (#SBATCH) and the seconrd part is the QIIME python subscript. Full details of the QIIME scripts used in subsequent analysis can be found on the QIIME website (http://qiime.org).  Scripts were created in .pbs files and submitted using the code 'sbatch'. 
